frontend/src/components/c-ramp.vue

```diff
@@ -2,7 +2,7 @@
 .ramp
   template(v-if="tframe === 'commit'")
     template(v-for="(slice, j) in user.commits")
-      template(v-for="(commit, k) in slice.commitResults.filter(commitResult => commitResult.insertions > 0)")
+      template(v-for="(commit, k) in slice.commitResults")
```

Comment:
Regarding this, I understand that the change here is intended to include merge commits. However, I am thinking about whether it is suitable to design merge commits to be empty. As the example given in #389 suggests, there can be certain merge requests that introduce additional changes when resolving conflicts. This means it may be necessary to show the number of additions and deletions for merge commits on the dashboard as well for consistency reasons.

Reply:
Sorry, I'm not sure what you mean by "designing merge commits to be empty". 

Since even empty merge commits can include information like tags, I have included them in the chart. I believe I am displaying the number of additions and deletions, irrespective of the type of commit, for both the ramp chart and the commits panel. 

For the highlighted line in the code, are you suggesting we filter using: 
`.filter(commitResult => this.getContributions(commitResult) > 0 || commitResult.isMergeCommit)`?

Reply:
Oh I was under the impression that merge commits will be empty cos you mentioned above that these commits will only not be shown if any file type is excluded. If fileTypeAndContributionMap and number of additions, deletions work the same way as other commits, then you can just ignore my comment. Thanks

Reply:
Oh, what I meant to say is that *empty* merge commits will not be shown when any file type is excluded, since they do not come under any file type category. 

When all file types are selected, I bypass the filtering of commits by the selected file types and display all commits, so that empty merge commits (that do not come under any file type) are included.

Reply:
Thanks for the clarification. Actually, what caused my confusion is that while the merge commit in https://github.com/Jnjy/ip/commit/d847cd09cfe4c4591e960d77e5b9b6bc70d97fce has a positive number of additions and deletions, the screenshot above shows otherwise ...... If that is intended, please go ahead

Reply:
According to the discussion in the lecture, let's add the following changes to the commits panel

1. Add a Merge label next to each commit title in the commits panel
2. In addition to the numerical line counts, add a contribution bar for each commit in the commits panel, similar to that in the chart panel, to indicate the number of additions and deletions. Optionally, color the addition bar to be green and the deletion bar to be red.

Reply:
I've added the `code-merge` icon beside the titles of merge commits.